frame/composable-traits/src/vault.rs

```diff
@@ -27,6 +28,18 @@ pub enum FundsAvailability<Balance> {
 	None,
 }
 
+#[derive(Copy, Clone, Encode, Decode, MaxEncodedLen, Default, Debug, PartialEq, TypeInfo, Eq)]
+pub struct StrategyOverview<Balance> {
```

Comment:
I do not know how the relationship between these files is, but `StrategyOverview` appears in the following files:
- [`composable_api.rs`](https://github.com/ComposableFi/composable/blob/2805639cd55364dee41217bf68dd28b1de489aad/utils/price-feed/src/feed/composable_api.rs),
- [`composablejs` directory](https://github.com/ComposableFi/composable/tree/pablo-feat/constant_product_zapping/composablejs),
- [`integration-tests` directory](https://github.com/ComposableFi/composable/tree/pablo-feat/constant_product_zapping/integration-tests/runtime-tests),
- [`frontend` directory](https://github.com/ComposableFi/composable/tree/pablo-feat/constant_product_zapping/frontend/packages/defi-interfaces).

Reply:
This was moved here from the vault pallet, so it already exists